wordpress-mobile/WordPress-iOS
# Animated gif support for fullscreen image from the media library item view

Fixes #9355 

This PR adds an `ImageLoader` to the `WPImageViewController.m` class to be used when loading a `Media` object directly.

This change made posible to play animated gifs from the `MediaItemViewController` full screen media preview.

![animated_gif_full_screen](https://user-images.githubusercontent.com/9772967/40125624-582b07f2-58f9-11e8-9da2-bb3eea5dc6ba.gif)

To test:
- Upload an animated gif to your WP Media Library.
- Open that gif media item in your Media Library.
- Tap on the animated image.
- Check that the full screen view is animating too.
